app/CsvImportModule/Contracts/CsvExtractedData.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace App\CsvImportModule;
+
+
+interface CsvExtractedData
+{
+    /**
+     * CsvExtractedData constructor.
+     * @param array $dataArray
+     * @param array $columnHeadersRowArray
+     */
+    function __construct(array $dataArray, array $columnHeadersRowArray = []);
+
+    /**
+     * @param array $dataArray
+     */
+    function setDataArray(array $dataArray): void;
```

Comment:
No need to name variables as `$dataArray` since it's data type is already array, it's more clear to simply use `$data`